content/news/2021-01-24-internet-censorship.md

```diff
@@ -0,0 +1,123 @@
+title: "About my Outreachy internship with Mixxx DJ: Contributing to open source amidst Uganda’s internet restrictions"
+authors: Aanyu Deborah Oduman
+tags: personal, outreachy, internship
+date: 2021-01-24 14:00:00
+summary: Last week, amid the 2021 general election, Uganda was disconnected from the internet entirely. The country experienced a widespread internet blackout that lasted 4 days, starting on the eve of the election (13th January 2021) and ending in the morning of 18th January 2021. In the days leading up to the election, access to major social media platforms and circumvention tools was blocked – even when the OTT (Over the Top) tax (commonly referred to as the “Social Media Tax”) was paid. This internet shutdown and the social media ban that followed affected many online jobs and businesses, my internship with Mixxx under the Outreachy program was no exception.
+
+![Internet shutdown]({static}/images/news/internet_shutdown.jpg)
+
+Before I write this article, one thing to note is that Ugandans have been in this internet battle with the government for quite some time now.
+The internet shut down came as a surprise when it did, but at the end of the day, I feel like we should have seen it coming.
+By the time it dawned on us that the internet really had been disconnected, we were already in darkness, with only the people physically close to us to express our shock too.
+The audacity of this government!
+
+As the years go by, our President, H.E Yoweri Kaguta Museveni, has been feeling a lot of pressure about the presidential seat.
+If elections were conducted freely and fairly, you would notice a declining trend in the number of votes his political party (NRM) has been getting over the years.
+Ugandans are tired, and they want change. They have been more vocal about the injustices that have been happening in the country and it has cost them their lives, even their families’.
+This resistance from the public has been causing anxiety in State House, and it’s starting to show.
+
+In the 2016 presidential elections, Ugandan authorities blocked access to Facebook, Twitter and WhatsApp saying the platforms would be used by the opposition to mobilize protests.
+(See [[this article from the BBC](https://www.bbc.com/news/world-africa-35601220))
```

Comment:
Maybe rephrase to "This article from the bbc has more context on the social media blockage"